server_test.go

```diff
@@ -117,10 +117,6 @@ func (h *testResourceHandler) Create(r *http.Request, attributes scim.ResourceAt
 }
 
 func (h *testResourceHandler) Delete(r *http.Request, id string) error {
-	if err := checkBodyNotEmpty(r); err != nil {
-		return err
-	}
-
```

Comment:
Is there a reason these are removed?

Reply:
Yes, on all the read endpoints these dont make sense since that will always be without a body